### ISSUE TITLE
Storefront: WCAG Errors

### ISSUE THREAD

**Issue by mrosvik** [digdir/designsystem#534]
2 errors found by Wave: 
- language missing
- empty button

![Image](https://github.com/digdir/designsystem/assets/1464915/6f5eb0cf-f956-469a-a809-30c037f0f876)

**Comment by mrosvik** [comment]
Tested wrong branch. This is fixed.
